README.md

```diff
@@ -1,4 +1,4 @@
-Djangosaml2 SPID
+SPID Django (former [djangosaml2_spid](https://github.com/peppelinux/djangosaml2_spid))
```

Comment:
only SPID Django please :)

Reply:
No problem! On it! 😎 👍🏻

Reply:
@peppelinux Done! 👍🏻 
Check it out and resolve the conversation.